include/xgboost/learner.h

```diff
@@ -186,15 +186,16 @@ class Learner : public rabit::Serializable {
    */
   virtual const std::map<std::string, std::string>& GetConfigurationArguments() const = 0;
 
+  /*! \brief The evaluation metrics used to evaluate the model. */
+  std::vector<std::unique_ptr<Metric> > metrics_;
```

Comment:
sorry for the late reply, just back from the vacation, because I referred metrics_ in https://github.com/dmlc/xgboost/pull/4280/files#diff-3288207540b433242fe8da1390c68d3fR1164